x-pack/plugins/security_solution/common/endpoint/service/authz/mocks.ts

```diff
@@ -16,7 +16,7 @@ export const getEndpointAuthzInitialStateMock = (
       Object.entries(getEndpointAuthzInitialState()) as Array<[keyof EndpointAuthz, boolean]>
     ).reduce((mockPrivileges, [key, value]) => {
       // Invert the initial values (from `false` to `true`) so that everything is authorized
-      mockPrivileges[key] = !value;
+      mockPrivileges[key] = true;
```

Comment:
Good question:

Its because we have been leaving FF around after they are enabled by default. So the prior code was actually setting feature that were by default se to `true` to `false` which is now what we want out of the initial mock state